src/mapstorage/mapstorage.cpp

```diff
@@ -955,6 +975,8 @@ void MapStorage::saveMark(InfoMark * mark, QDataStream & stream)
   stream << (QString)mark->getText();
   stream << (QDateTime)mark->getTimeStamp();
   stream << (quint8)mark->getType();
+  stream << (quint8)mark->getClass();
+  stream << (qint32)(mark->getRotation()*100);
```

Comment:
If you agree that a better name is `m_rotationAngle` then this should be changed to `getRotationAngle()` as well.